src/test/java/core/basesyntax/service/parse/ParserServiceTest.java

```diff
@@ -0,0 +1,57 @@
+package core.basesyntax.service.parse;
+
+import static org.junit.Assert.assertEquals;
+
+import core.basesyntax.model.FruitTransaction;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class ParserServiceTest {
+    private static ParserService parserService;
+
+    @BeforeClass
+    public static void setUp() {
+        parserService = new ParserServiceImpl();
+    }
+
+    @Test
+    public void parse_validCase_OK() {
+        List<String> input = new ArrayList<>();
+        input.add("operation,fruit,quantity");
+        input.add("b,apple,18");
+        input.add("b,banana,25");
+        input.add("p,banana,6");
+        List<FruitTransaction> expected = new ArrayList<>();
+        expected.add(new FruitTransaction("b",
+                "apple", 18));
+        expected.add(new FruitTransaction("b",
+                "banana", 25));
+        expected.add(new FruitTransaction("p",
+                "banana", 6));
+        List<FruitTransaction> actual = parserService.parse(input);
+        assertEquals(expected, actual);
+    }
+
+    @Test
+    public void parse_header_Ok() {
+        List<String> input = new ArrayList<>();
+        input.add("operation,fruit,quantity");
+        List<FruitTransaction> expected = Collections.EMPTY_LIST;
+        List<FruitTransaction> actual = parserService.parse(input);
+        assertEquals(expected, actual);
+    }
+
+    @Test (expected = IndexOutOfBoundsException.class)
+    public void parse_EmptyInput_NotOk() {
```

Comment:
```suggestion
    public void parse_emptyInput_NotOk() {
```
пересмотри названия всех тестовых методов во всех классах, и давай все названия приведем под один паттерн